n2survey/lime/survey.py

```diff
@@ -458,19 +462,89 @@ def plot(
         theme = deep_dict_update(theme, theme_kwargs)
 
         question_type = self.get_question_type(question)
+        # get plot title
+        if plot_title:
+            if type(plot_title) != str:
+                plot_title = self.get_label(question)
 
         if question_type == "single-choice":
             counts_df = self.count(question, labels=True)
 
             if "title" not in non_theme_kwargs:
                 non_theme_kwargs.update({"title": counts_df.columns[0]})
 
-            fig, ax = single_choice_bar_plot(
-                x=counts_df.index.values,
-                y=pd.Series(counts_df.iloc[:, 0], name="Number of Responses"),
-                theme=theme,
-                **non_theme_kwargs,
-            )
+            if compare_with:
+                DF = []
```

Comment:
yep, will change that. 
Just skimmed some guides for proper programming and somewhere I read for global variables it is good to be Uppercase, because you could distinguish them easier from functions, but I guess now as soon as the variable is in a function/method it would de facto not be a global one any more.